student.rb

```diff
@@ -0,0 +1,12 @@
+class Student < Person
+  # constructor
+  def initialize(classroom)
+    super(id, age, name, parent_permission: true)
+    @classroom = classroom
+  end
```

Comment:
- [ ] Good job create the Student Class that inherits from the person Class 👏  , however as require , The constructor should take four parameters - age, classroom, name, and parent_permission. The age and classroom parameters are mandatory, while name has a default value of "Unknown," and parent_permission has a default value of true. This allows for flexibility when creating instances of the class. Kindly update the Student Constructor 👍 

```ruby
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```

Reply:
Really good review, I changed it due to a friend's suggestion, and now I am going to pass the voice to properly initialize classes that have inheritances, thanks for the amazing review and suggestions. 😁